src/api/python/hidra/utils/utils_general.py

```diff
@@ -394,6 +404,13 @@ def close(self):
 
 
 def open_tempfile(filepath):
+    """
+    Open a temporary file that will be automatically renamed to filepath when
+    closed.
+
+    The file is opened in binary write mode.
```

Comment:
Since the default mode is w+b I think reads are also possible...

Reply:
I don't have any opinion on the wrapper itself, but I would match the mode with the usage. It could turn out that you do need w+, so I am not saying it's wrong, BTW.

Reply:
Done